runtime_lib/airhost/CMakeLists.txt

```diff
@@ -12,57 +12,65 @@ include_directories(
 add_definitions(-DLIBXAIENGINEV2)
 
 # Only build the runtime if hsa was found
-if (hsa-runtime64_FOUND)
-  include_directories(${XILINX_XAIE_INCLUDE_DIR})
-  link_libraries(${XILINX_XAIE_LIBS})
+if (hsa-runtime64_FOUND AND TARGET hsa-runtime64::hsa-runtime64)
+  # Optionally validate that the library file exists
+  get_target_property(HSA_RUNTIME_LIB hsa-runtime64::hsa-runtime64 IMPORTED_LOCATION)
+  if (EXISTS "${HSA_RUNTIME_LIB}")
 
-  add_library(airhost STATIC
-      memory.cpp
-      queue.cpp
-      runtime.cpp
-      host.cpp
-      pcie-ernic.cpp
-      pcie-ernic-dev-mem-allocator.cpp
-      network.cpp
-  )
-  set_property(TARGET airhost PROPERTY POSITION_INDEPENDENT_CODE ON)
+    include_directories(${XILINX_XAIE_INCLUDE_DIR})
+    link_libraries(${XILINX_XAIE_LIBS})
```

Comment:
I don't understand what's going on here. `hsa-runtime64_FOUND` should only be set iff the library was found. i.e. "library file exists"